docs/pages/components/busy-indicator.md

```diff
@@ -17,16 +17,16 @@ Busy indicators are not visible all the time, only when needed.
 ## Busy indicator usage and size options
 {% capture busy-indicator %}
 <div style="display:flex;justify-content:center;flex-direction:column;align-items:center">
-    <div class="fd-busy-indicator--l" aria-hidden="false" aria-label="Loading">
+    <div class="fd-busy-indicator--l" aria-hidden="false" aria-label="Loading"  style="margin-bottom:2rem">
```

Comment:
We should not apply styling like this. You are adding margin-bottom to the element (`fd-busy-indicator--l`). <br> is not part of the element markup, it was used to render the busy indicator on a new line. You can add styling to the documentation (the margin) in the doc style file